src/deepMergeLeft.ts

```diff
@@ -0,0 +1,27 @@
+import { recursiveMerge, DeepPartialObject } from './_recursiveMerge';
+
+/**
+ * Merging object from left to right
+ *
+ * @description
+ * @param target value be preserved if possible.
+ * @param source
+ * Consider following
+ *
+ * array + obj = array
+ * obj + array = obj
+ * obj + obj = obj (recursively merged)
+ * array + array = array (removes duplicates using Set)
+ * (truthy plain value) + ob = (truthy plain value)
+ * (truthy plain value) + undefined = (truthy plain value)
+ * A(truthy plain value) + B(truthy plain value) = A(truthy plain value)
+ *
+ * Handles circular references
+ * @returns {T}
+ */
+export function deepMergeLeft<T extends object>(
+  target: T,
+  source: DeepPartialObject<T>
+): T {
+  return recursiveMerge(target, source);
+}
```

Comment:
This simply calls the recursiveMerge function, then why do we need it? simply use the recursiveMerge function. I would simply named it deepMerge and drop the left/right concept. It's enough to have one direction, and anyone who needs the other direction can simply call the function with the params reversed.

Also, please follow the format we have for other files and create the data-first and data-last signatures and use purry to call the implementation

Reply:
done